src/app/migrations/0002_auto_20230529_1433.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.18 on 2023-05-29 14:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='licensenamespace',
+            name='isException',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AddField(
+            model_name='licensenamespace',
+            name='text',
+            field=models.TextField(default=''),
```

Comment:
You will need to commit only changed migrations if this is fixed in the other PR